client/my-sites/checklist/wpcom-checklist/wpcom-task-list.js

```diff
@@ -54,8 +54,14 @@ export default class WpcomTaskList {
 			addTask( 'site_launched' );
 		}
 
-		if ( config.isEnabled( 'onboarding-checklist/email-setup' ) && hasTask( 'email_setup' ) ) {
-			addTask( 'email_setup' );
+		if ( config.isEnabled( 'onboarding-checklist/email-setup' ) ) {
+			if ( hasTask( 'email_setup' ) ) {
+				addTask( 'email_setup' );
+			}
+
+			if ( hasTask( 'upgrade_email_forwarding' ) ) {
+				addTask( 'upgrade_email_forwarding' );
```

Comment:
Shouldn't we name this task `email_forwarding_upgraded` to be consistent with other tasks?